manifests/chrome.json

```diff
@@ -1,47 +1,44 @@
 {
   "version": "1.1.0",
```

Comment:
Please update version here accordingly here and for Firefox

Reply:
the version is updated by the release script

Reply:
So we should remove it from there (as we have in main extension)